### PR TITLE
add campaign info to docs

### PR DESCRIPTION
**Pull request by twessling-icas**

### CODE REVIEW

File: doc/next/_sources/campaigns.rst.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+======================== ================================================
+Component                Short description
+======================== ================================================
+:ref:`campaign_id`       The unique reference to the Campaign
+:ref:`campaign_title`    User-provided title
+:ref:`campaign_vendorid` Unique reference for the seller to the campaign
+:ref:`campaign_status`   Status of the campaign
+:ref:`campaign_links`    Ads in this campaign
+:ref:`campaign_budgets`  Budgets of this campaign
```
Comment (remenska) [inline doc/next/_sources/campaigns.rst.txt]: targeting  and cpc are also in

File: doc/next/_sources/campaign-budgets.rst.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.. index:: single: Campaign Budgets
+.. _campaign_budgets_overview:
+
+Campaign Budgets
+================
+
+Each campaign can have 0 or more budgets. Without budgets, the ads in the campaign will not be shown.
```
Comment (remenska) [inline doc/next/_sources/campaign-budgets.rst.txt]: We should stress that budgets are merely limitations on the campaign spending. Once any budget is set, all of the rest of the budget types are assumed to be unlimited. Deleting budgets is not really a way to ensure teh campaign content will not be live (campaign status is for this), this just means the budgets are unlimited. At least that was the conclusion from our discussion.

File: doc/next/_sources/campaign-status.rst.txt [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ :header-rows: 1
+
+ * - Status
+   - Description
+
+ * - **BUDGET_REACHED**
+   - One or more of the campaign's budgets have been reached and its linked ads are not shown on site. In case of recurring budgets, like Daily or Monthly budgets, the budgets will be periodically automatically reset and the campaign will become active again. This status has a precedence only over the user-controlled status ``ACTIVE``; in other words, if the campaign is ``PAUSED`` (or ``DELETED``), while a campaign budget is reached (``BUDGET_REACHED``), the campaign will be shown as ``PAUSED`` (or ``DELETED``).
+
+
+.. image:: _static/CampaignStates.png
```
Comment (fatihtekin) [inline doc/next/_sources/campaign-status.rst.txt:52]: should there be a line from BUDGET_REACHED to PAUSED as well?

File: doc/next/_sources/campaign-linkedads.rst.txt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+.. _campaign-linked-ads:
+
+
+Campaign's linkedAds component defines which ads are part of the campaign. It is possible to add / remove (link / unlink) ads from a campaign using
+:ref:`put_campaign_linkedads` / :ref:`delete_campaign_linkedads`. Deleting a linkedAd from a campaign does not remove the ad from your inventory, it merely takes it offline in the context of the particular campaign.
+
+LinkedAds are stored as part of the campaign, and have a reference to an Ad using its ID. A linkedAd consists of two parts:
+the Ad ID and optional overrides from the campaign defaults.
+Using linkedAds, it is possible to add a single ad to multiple campaigns (which may have different budgets and targeting criteria).
```
Comment (fatihtekin) [inline doc/next/_sources/campaign-linkedads.rst.txt]: should we also mention ad can be in multiple campaigns as  `(future plans)` like mentioned in campaign-overview.rst.txt  L72

File: doc/next/_sources/reporting.rst.txt [hunk 5]
```diff
@@ -262,6 +263,7 @@ Name                           Type        Description
 ``am:currentAdVendorID``      String       Current vendorID of the ad, if available
 ``am:currentAdRegion``        String       Description of the current (lowest-level) region of the ad, if applicable
 ``am:currentAdExternalID``    String       Current externalID of the ad, if available
+``am:currentCampaignID``      Long         Current campaignID to which the ad belongs // TODO: need to check if this makes sense in general
```
Comment (twessling-icas) [inline doc/next/_sources/reporting.rst.txt:266]: I don't think this one makes sense, if you're getting data on ad basis.

**Review by homiak — DISMISSED** [review]
Nice update!

**Comment by twessling-icas** [comment]
superseded by #65

**Pull request closed without merging**